mlos_core/mlos_core/spaces/adapters/__init__.py

```diff
@@ -7,28 +7,38 @@
 """
 
 from enum import Enum
-from typing import Optional, TypeVar
+from typing import Optional, TypeVar, Union
 
 import ConfigSpace
 
+from mlos_core.spaces.adapters.adapter import BaseSpaceAdapter
+from mlos_core.spaces.adapters.identity_adapter import IdentityAdapter
 from mlos_core.spaces.adapters.llamatune import LlamaTuneAdapter
 
 __all__ = [
+    'IdentityAdapter',
     'LlamaTuneAdapter',
 ]
 
 
 class SpaceAdapterType(Enum):
     """Enumerate supported MlosCore space adapters."""
 
-    IDENTITY = None
+    IDENTITY = IdentityAdapter
```

Comment:
mypy is happier having an explicit type here, so we add the Identity one and tests for it.